src/consensus/validator.rs

```diff
@@ -53,12 +82,18 @@ impl ShardValidator {
         }
     }
 
-    pub fn get_validator_set(&self) -> SnapchainValidatorSet {
-        self.validator_set.clone()
+    pub fn get_validator_set(&self, height: u64) -> SnapchainValidatorSet {
+        let mut result = &self.validator_set[0];
```

Comment:
We might need to panic somewhere above if validator set length is 0. Preferably as early as possible (in the config, within validator_set_for?)

Reply:
Done in https://github.com/farcasterxyz/snapchain/pull/279/commits/b7f04b5c674698afe47f34029a058e91996e578f